### PR TITLE
CLN: assorted cleanups

### PR DESCRIPTION
**Pull request by jbrockmendel**
broken off from other branches

### CODE REVIEW

File: pandas/core/internals/managers.py [hunk 1]
```diff
@@ -584,6 +584,7 @@ def convert(self, **kwargs):
         return self.apply("convert", **kwargs)
 
     def replace(self, **kwargs):
+        assert "value" in kwargs and np.ndim(kwargs["value"]) == 0
```
Comment (WillAyd) [inline pandas/core/internals/managers.py]: Does adding this to the signature explicitly not work?

File: pandas/tests/computation/test_eval.py [hunk 2]
```diff
@@ -411,7 +411,7 @@ def check_single_invert_op(self, lhs, cmp1, rhs):
                 )
 
     def check_compound_invert_op(self, lhs, cmp1, rhs):
-        skip_these = "in", "not in"
+        skip_these = ["in", "not in"]
```
Comment (WillAyd) [inline pandas/tests/computation/test_eval.py]: Maybe make a tuple for consistency with old code?

File: pandas/core/internals/managers.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/core/internals/managers.py]: I think it does, but the pattern for the other self.aply usages here is just to have kwargs.  I'm OK either way

File: pandas/tests/computation/test_eval.py (hunk 2 above)
Comment (jbrockmendel) [reply pandas/tests/computation/test_eval.py]: Just used for an inclusion check; either way works.

File: pandas/core/internals/managers.py (hunk 1 above)
Comment (WillAyd) [reply pandas/core/internals/managers.py]: Yea I think signature would be better if required

Comment (jbrockmendel) [reply pandas/core/internals/managers.py]: changed

**Comment by jreback** [comment]
thanks @jbrockmendel

**Pull request merged**